src/cr/cube/crunch_cube.py

```diff
@@ -237,7 +237,7 @@ def _as_array(self, include_missing=False, get_non_selected=False,
 
         if prune and not self.has_mr:
             # Remove columns and rows where marginal value is 0 or np.nan
-            if len(self.dimensions) == 1:
+            if len(self.dimensions) == 1 or len(res.shape) == 1:
```

Comment:
hmm. It’s tricky where to decide to reshape `(1)` to `(1, )`. 
I wonder if there’s a way to do it earlier so as to avoid an extra condition here. This fix is super simple so 👍 but just thinking about factoring the cube in the future.